pkg/auth/providers/azure/clients/ms_graph_client.go

```diff
@@ -112,11 +114,42 @@ func (c azureMSGraphClient) ListGroupMemberships(id string) ([]string, error) {
 func (c azureMSGraphClient) LoginUser(config *v32.AzureADConfig, credential *v32.AzureADLogin) (v3.Principal, []v3.Principal, string, error) {
 	logrus.Debugf("[%s] Started token swap with AzureAD", providerLogPrefix)
 
-	// Acquire the OID just to verify the user.
-	oid, err := oidFromAuthCode(credential.Code, config)
-	if err != nil {
-		return v3.Principal{}, nil, "", err
+	var oid string
+	if credential.IDToken != "" {
+		issuer := fmt.Sprintf("%s%s/v2.0", config.Endpoint, config.TenantID)
```

Comment:
Again, you can use `url.JoinPath` for this for a bit more confidence over the resulting value.

Reply:
Done with 138b331